app/src/main/java/dev/arkbuilders/arkmemo/models/GraphicNote.kt

```diff
@@ -0,0 +1,17 @@
+package dev.arkbuilders.arkmemo.models
+
+import android.os.Parcelable
+import dev.arkbuilders.arkmemo.data.ResourceMeta
+import dev.arkbuilders.arkmemo.utils.SVG
+import kotlinx.parcelize.IgnoredOnParcel
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class GraphicNote(
+    val title: String,
+    val description: String = "",
+    val content: Content,
```

Comment:
Is it a duplicate of the data stored in `svg`? I think we should just implement `Parcelable` for `SVG` type.

Reply:
You are right, that was a quick solution for prototype. We could just use ```SVG```